src/azul/__init__.py

```diff
@@ -782,6 +788,10 @@ def dynamo_object_version_table_name(self) -> str:
 
     terms_aggregation_size = 99999
 
+    @property
+    def tracked_schema_path(self) -> str:
+        return os.path.join(self.project_root, 'terraform/_schema.json')
```

Comment:
We prefer `pathlib.Path`.

Reply:
To clarify, does this mean that I should return `tracked_schema_path` as pathlib.Path, I should use pathlib.Path instead of os.path.join, or both?

Reply:
(The first one.)

Reply:
We are moving to pathlib. When modifying an existing module the choice of whether to update the module to pathlib or to post-pone that and stick with os.path withing the module is at the discretion of the author. The decision should be guided by the urgency of the issue at hand vs the complexity of the update.